app/forms/select_email_form.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class SelectEmailForm
+  include ActiveModel::Model
+  include ActionView::Helpers::TranslationHelper
+
+  attr_reader :user, :selected_email
+
+  validate :validate_owns_selected_email
+
+  def initialize(user)
+    @user = user
+  end
+
+  def submit(params)
+    @selected_email = params[:selection]
```

Comment:
Similar to a [previous review comment](https://github.com/18F/identity-idp/pull/10951#discussion_r1688822544), I think it would help make things clearer to clearly name this as being an ID. It might also help to use consistent naming, rather than `selected_email` vs. `selection`.

```suggestion
    @selected_email_id = params[:selected_email_id]
```

Reply:
standardized the variables for consistency and clarity